packages/connect-popup/src/view/selectDevice.ts

```diff
@@ -47,8 +67,11 @@ const initWebUsbButton = (showLoader: boolean) => {
                 throw ERRORS.TypedError('Popup_ConnectionMissing');
             }
         } catch (error) {
-            // empty, do nothing, should not happen anyway
             console.error(error);
+            reactEventBus.dispatch({
+                type: 'error',
+                detail: 'iframe-failure',
+            });
```

Comment:
this is to prevent some endless loader?

Reply:
No previously the error was silent (only logged to console), I think it's better to display the error

Reply:
It will render the Error component from connect-ui